python/orca/src/bigdl/orca/learn/pytorch/experimential/mmcv/mmcv_ray_runner.py

```diff
@@ -162,23 +162,25 @@ def run_iter(self, data_batch: Any, train_mode: bool, **kwargs) -> None:
         elif train_mode:
             outputs = self.model.train_step(data_batch, self.optimizer,
                                             **kwargs)
-            if kwargs.get("metric_meters"):
-                metric_meters = kwargs["metric_meters"]
-                copy = dict()
-                for k, v in outputs.items():
-                    if isinstance(v, torch.Tensor):
-                        copy[k] = v.item()
-                    else:
-                        copy[k] = v
-                metric_meters.update(copy, n=len(data_batch))
         else:
             outputs = self.model.val_step(data_batch, self.optimizer, **kwargs)
         if not isinstance(outputs, dict):
             invalidInputError(False,
                               '"batch_processor()" or "model.train_step()" '
                               'and "model.val_step()" must return a dict')
+
         if 'log_vars' in outputs:
-            self.log_buffer.update(outputs['log_vars'], outputs['num_samples'])
+            if 'loss' in outputs['log_vars']:
```

Comment:
make some simple comments here like `ensure key loss is in log_buffer, and driver can fetch loss infos from workers' log_buff`